WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift

```diff
@@ -102,6 +115,7 @@ class SiteStatsInsightsTableViewController: UITableViewController, StoryboardLoa
     private var allSitesInsights = [SiteInsights]()
     private typealias SiteInsights = [String: [Int]]
 
+    private var displayingEmptyView = false
```

Comment:
`displayingEmptyView` isn't to indicate if the NRV is showing. It's to know which NRV is showing.

Since there are two possible NRVs with an action button (empty view and loading failure), when `actionButtonPressed` is called, the action depends on which NRV is being displayed.